src/supervisor/workload-reader.ts

```diff
@@ -253,13 +252,10 @@ const cronJobReader: IWorkloadReaderFunc = async (workloadName, namespace) => {
     .retryKubernetesApiRequest(() =>
       k8sApi.batchClient.readNamespacedCronJob(workloadName, namespace),
     )
-    // In case the V1 client fails, try using the V1Beta1 client.
     .catch(() =>
-      kubernetesApiWrappers.retryKubernetesApiRequest(() =>
-        k8sApi.batchUnstableClient.readNamespacedCronJob(
-          workloadName,
-          namespace,
-        ),
+      logger.info(
+        { workloadName, namespace },
+        'kubernetes api could not return workload',
```

Comment:
not sure on this log, in the jobReader above we do not catch the error, lets follow that pattern?